src/crafting.cpp

```diff
@@ -1882,3 +1913,16 @@ const std::string &recipe::ident() const
 {
     return ident_;
 }
+
+std::vector<npc *> player::get_crafting_helpers() const
+{
+    std::vector<npc *> ret;
+    for( auto &elem : g->active_npc ) {
+        if( rl_dist( elem->pos(), pos() ) < PICKUP_RANGE && elem->is_friend() &&
```

Comment:
LoS checks don't protect from that - reinforced windows, vehicles etc. would still prevent it.
Accessibility check, like for tools, could help, though.